src/test/java/io/appium/java_client/android/AndroidAbilityToUseSupplierTest.java

```diff
@@ -19,7 +19,7 @@
 
 public class AndroidAbilityToUseSupplierTest extends BaseAndroidTest {
 
-    private final ActionSupplier<AndroidTouchAction> horizontalSwipe = () -> {
```

Comment:
Please revert changes to this test. We will remove it as soon as deprecated interfaces are removed